content/en/news/releases/1.13.x/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 1.13.x Releases
+description: Announcements for the 1.13 release and its associated patch releases.
+weight: 17
```

Comment:
16 please, 1.12 has the wrong weight and will be fixed in a different PR

Reply:
```suggestion
weight: 16
```